app/assets/javascripts/discourse/app/controllers/composer.js

```diff
@@ -1008,7 +1008,9 @@ export default Controller.extend({
     // TODO: This should not happen in model
     const imageSizes = {};
     document
-      .querySelectorAll("#reply-control .d-editor-preview img")
+      .querySelectorAll(
+        "#reply-control .d-editor-preview img:not(.avatar):not(.emoji)"
```

Comment:
I think that works as well?

```suggestion
        "#reply-control .d-editor-preview img:not(.avatar, .emoji)"
```

Reply:
Naah, that doesn't work, the pseudoclass doesn't take multiple classes: https://css-tricks.com/css-not-with-multiple-classes/.

Reply:
That's not what [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/:not#description) says 🤔 


> You can negate several selectors at the same time. Example: `:not(.foo, .bar)` is equivalent to `:not(.foo):not(.bar)`.

Reply:
You are right, TIL, this now does work. Initially it didn't, per: https://caniuse.com/css-not-sel-list